textattack/transformations/word_swap_inflections.py

```diff
@@ -35,7 +35,7 @@ def _get_replacement_words(self, word, word_part_of_speech):
 
     def _get_transformations(self, current_text, indices_to_modify):
         words = current_text.words
-        sentence = Sentence(current_text.text)
+        sentence = Sentence(" ".join(words))
```

Comment:
Why this change?

Reply:
FLAIR applies part-of-speech tags to punctuation and bits of punctuation. Without this, the part-of-speech tags wouldn't line up with `current_text.words` (and an error would be thrown in special cases)

Reply:
Hmm can it assign the right POS to words without punctuation though?

Reply:
yeah, this is a good concern, I shall put it on Asana